src/model/NPMResolver.ts

```diff
@@ -164,6 +165,7 @@ export class NPMResolver {
     }
 
     public async resolve(): Promise<void> {
+        this.initRegistry();
```

Comment:
Do we want that on the project level? Or maybe the plugin level?

Reply:
It's on npm user level that he credentials are added